### PR TITLE
SE-1703: Add TCP keep alive changes to required templates

### PR DESCRIPTION
**Pull request by fightingsleep**
There is a corresponding pull request in the python SDK [here](https://github.com/finbourne/lusid-sdk-python-preview/pull/118). Please see its description as well.

The changes here are to add a configuration.mustache template which will contain the new TCP keep alive option. This template will be used to generate [configuration.py](https://github.com/finbourne/lusid-sdk-python-preview/blob/master/sdk/lusid/configuration.py) which is used in the Python SDK. Similarly, the rest.mustache template has been updated to contain TCP keep alive functionality (a flag to control if the TCP keep alive version of the pool manager is used). This rest.mustache template is used to create [rest.py](https://github.com/finbourne/lusid-sdk-python-preview/blob/master/sdk/lusid/rest.py) in the python SDK.

### CODE REVIEW

File: python/generate/default_templates/configuration.mustache [hunk 1]
```diff
@@ -0,0 +1,626 @@
+{{>partial_header}}
```
Comment (MikeMcGarry) [inline python/generate/default_templates/configuration.mustache:1]: Was this template the correct Python flavour and version? It seems to have led to unexpected diffs in the SDK merge request.

File: python/generate/templates/rest.mustache [hunk 2]
```diff
@@ -74,28 +74,53 @@ class RESTClientObject(object):
             else:
                 maxsize = 4
 
+        # tcp_keep_alive cannot be put in additional_pool_args without errors in the base pool manager
```
Comment (MikeMcGarry) [inline python/generate/templates/rest.mustache:77]: Good to know

File: python/generate/templates/configuration.mustache [hunk 3]
```diff
@@ -0,0 +1,629 @@
+{{>partial_header}}
```
Comment (MikeMcGarry) [inline python/generate/templates/configuration.mustache:1]: I am assuming that the changes you have introduced here are reflected in the SDK merge request>?

Comment (fightingsleep) [reply python/generate/templates/configuration.mustache:1]: Correct, this has the exact same changes. Let me know which order we typically merge these in. Do I merge the lusid-sdk-python-preview changes first, and then these changes? Or vice versa

File: python/generate/default_templates/configuration.mustache (hunk 1 above)
Comment (fightingsleep) [reply python/generate/default_templates/configuration.mustache:1]: Quite possibly not, I took the latest one from their master branch. Let me reconsider

Comment (MikeMcGarry) [reply python/generate/default_templates/configuration.mustache:1]: Great thanks, you can use the tags to find the correct version.

File: python/generate/templates/configuration.mustache (hunk 3 above)
Comment (MikeMcGarry) [reply python/generate/templates/configuration.mustache:1]: We will probably only commit the SDK changes without the new generated files and then make the generator change afterwards.

**Review by MikeMcGarry — APPROVED** [review]

**Pull request merged**